cmd/kyma/apply/function/function.go

```diff
@@ -122,6 +131,22 @@ func (c *command) Run() error {
 	return mgr.Do(ctx, options)
 }
 
+func (c *command) kymaHostAddress() string {
+	var url string
+	vs, err := c.K8s.Istio().NetworkingV1alpha3().VirtualServices("kyma-system").Get(context.Background(), "apiserver-proxy", v1.GetOptions{})
```

Comment:
As far as I know, `apiserver-proxy` is no longer going to be available soon and this code will stop working.
There are many alternatives to obtain the kyma host from; I would suggest getting it from `kyma-gateway` like this (kubectl example):
`kubectl get gateway kyma-gateway -n kyma-system -o yaml`

Reply:
I will investigate this. Thanks for this information! In this case, I'm getting the address from the apiserver-proxy virtualservice because this service is related to apiRules that this PR is about

Reply:
Yes it is not wrong, the issue is that `apiserver-proxy` will disappear soon and this code will fail.

We can just make it correctly long term now instead of waiting for it to fail.
It is only changing this one line with the gateway and we are good to go, the rest is all fine.

Reply:
Done. Thanks a lot!